cdk/cdk-resources/encryption-at-rest/package.json

```diff
@@ -17,14 +17,10 @@
     "eslint": "npx projen eslint",
     "package": "npx projen package",
     "package-all": "npx projen package-all",
-    "package:dotnet": "npx projen package:dotnet",
-    "package:java": "npx projen package:java",
     "package:js": "npx projen package:js",
-    "package:python": "npx projen package:python",
     "post-compile": "npx projen post-compile",
     "post-upgrade": "npx projen post-upgrade",
     "pre-compile": "npx projen pre-compile",
-    "publish:git:INTMDB-548": "npx projen publish:git:INTMDB-548",
     "release:INTMDB-548": "npx projen release:INTMDB-548",
```

Comment:
For my own knowledge, why does the issue name get used in the release target naming?

I would expect a common release target with a version generated

Reply:
This isn't the issue name, "INTMDB-548" is the branch name currently being used as default

Reply:
@SuperMohit Does this really matter since currently we're publishing manually? I guess this is relevant when using github actions to publish?